README.md

```diff
@@ -1,56 +1,44 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Book Store
 
-# Project Name
-
-> Description the project.
+## Screenshot ![see](./screenshot.png?raw=true 'Title')
 
+> The app retrieves available tickers from the API lists them in an home page. When a particular ticker is clicker it routes to the information of that ticker.
```

Comment:
- [ ] (Optional) The screenshot is not displaying professionally and it's making the `README.md` look unprofessional. I think it would be nice to either remove it or provide the access to the correct image you wish to use. The syntax looks like this:

```
![screenshot](./src/assets/Snapshot1.png)
```

If you decide to display the screen shot, you can use the syntax above in your `README.md` file. 👍